tests/pinkopy/test_helper.py

```diff
@@ -0,0 +1,52 @@
+import requests_mock
+import uuid
+
+
+def get_uuid():
+    return str(uuid.uuid4())
+
+
+def mock_session(base_session, service='http://' + get_uuid(),
+                 user=get_uuid(), pw=get_uuid(), token=get_uuid(),
+                 content_type='application/json', clients=get_uuid()
+                 ):
+    get_payload = {
+        'App_GetClientPropertiesResponse': {
+            'clientProperties': clients
+        }
+    }
+    post_payload = {
+        'DM2ContentIndexing_CheckCredentialResp': {
+            '@token': token
+        }
+    }
+    headers = {
+        'Accept': content_type,
+        'Content-type': content_type
+    }
+    with requests_mock.mock() as m:
+        m.get(service + '/Client', json=get_payload)
+        m.post(service + '/Login', headers=headers, json=post_payload)
```

Comment:
Yeah here, too. The `post_payload` should be `post_response` yielded from `m.post(urljoin(service, 'Login'), json=post_payload)` where `post_payload` is something along the lines of:
```python
payload = {
    'DM2ContentIndexing_CheckCredentialReq': {
        '@mode': 'Webconsole',
        '@username': user,
        '@password': b64encode(pw.encode('UTF-8')).decode('UTF-8')
    }
}
```
But that misses the fact that the json login is broken so there is a second use case where `m.post(urljoin(service, 'Login'), data=post_payload)` should yield similarly for:
```python
payload_nondict = ('<DM2ContentIndexing_CheckCredentialReq mode="Webconsole" '
                    'username="{}" password="{}" />'
                    .format(user, b64encode(pw.encode('UTF-8')).decode('UTF-8')))
```
if that response it parsed with xmltodict.